tools/pelican/content/plots.md

```diff
@@ -24,8 +24,9 @@ available for [every country](world.html).
 ![south-korea data]({attach}fig-south-korea2.png)
 
 * Shows how many new confirmed cases are reported per day, shown as blue bars.
-* Blue lines shows a seven day (rolling) average over the bar data - this
-  produces smoother data, and in particular removes the effect of the weekend
+* Blue lines shows a seven day (rolling) average over the bar data (using a Gaussian
+  window with a standard deviation of 3 days) - this
```

Comment:
Sometimes it's 2 days, not 3.

Reply:
Yes, that's for the growth factor visual smoothing. I'll try to come up with something better after this merge request.

For example, we could point to the source code and function name in the documentation.